include/zephyr/device.h

```diff
@@ -161,7 +161,7 @@ typedef int16_t device_handle_t;
  * @param api_ptr Pointer to the device's API structure. Can be NULL.
  */
 #define DEVICE_DEFINE(dev_name, drv_name, init_fn, pm_device,		\
-		      data_ptr, cfg_ptr, level, prio, api_ptr)		\
+			  data_ptr, cfg_ptr, level, prio, api_ptr)		\
```

Comment:
Could you remove all the unnecessary white-space changes from this PR? It would make it much easier to review.

Reply:
Done, whitespace differences don't show up in git diff by default apparently :)